app/webpacker/scripts/autocomplete.js

```diff
@@ -50,7 +50,7 @@ const initAutocomplete = ({element, input, path}) => {
         minLength: 3,
         source: throttle(request(path), 500),
         templates: {
-          inputValue: inputValueTemplate,
+          inputValue: selectNameAndCode ? suggestionTemplate : inputValueTemplate,
```

Comment:
This looks good @stevehook - regarding your query around testing - would adding an edge case here work?
https://github.com/DFE-Digital/find-teacher-training/blob/master/spec/features/new_results_page_filters/location_and_provider_spec.rb#L26

Reply:
@TobyRet I don't think I've changed anything in this PR that this kind of test covers. My concerns are testing my assumptions about the TTAPI (i.e. passing _Provider name (code)_ returns an exact match) and whether the JS changes work.

I think as we talked about on the #680 what we really need is some JS enabled feature specs to exercise the full stack (and ideally the TTAPI too).

Reply:
Yep ok agree 👍